src/model/server.rs

```diff
@@ -73,103 +41,219 @@ pub enum Server {
-            "ANIUMA" => Ok(Server::Aniuma),
-            "ASURA" => Ok(Server::Asura),
-            "BELIAS" => Ok(Server::Belias),
-            "CHOCOBO" => Ok(Server::Chocobo),
-            "HADES" => Ok(Server::Hades),
-            "IXION" => Ok(Server::Ixion),
-            "MANDRAGORA" => Ok(Server::Mandragora),
-            "MASAMUNE" => Ok(Server::Masamune),
-            "PANDAEMONIUM" => Ok(Server::Pandaemonium),
-            "SHINRYU" => Ok(Server::Shinryu),
-            "TITAN" => Ok(Server::Titan),
+
+       let fixed_world_name: &str = &*s.split_whitespace().next().unwrap();
+        match &*fixed_world_name.to_uppercase() {
+            
+            /// NA
             //  Aether
             "ADAMANTOISE" => Ok(Server::Adamantoise),
-            "BALMUNG" => Ok(Server::Balmung),
             "CACTUAR" => Ok(Server::Cactuar),
-            "COEURL" => Ok(Server::Coeurl),
             "FAERIE" => Ok(Server::Faerie),
             "GILGAMESH" => Ok(Server::Gilgamesh),
-            "GOBLIN" => Ok(Server::Goblin),
             "JENOVA" => Ok(Server::Jenova),
-            "MATEUS" => Ok(Server::Mateus),
             "MIDGARDSORMR" => Ok(Server::Midgardsormr),
             "SARGATANAS" => Ok(Server::Sargatanas),
             "SIREN" => Ok(Server::Siren),
-            "ZALERA" => Ok(Server::Zalera),
+
             //  Primal
             "BEHEMOTH" => Ok(Server::Behemoth),
-            "BRYNHILDR" => Ok(Server::Brynhildr),
-            "DIABOLOS" => Ok(Server::Diabolos),
             "EXCALIBUR" => Ok(Server::Excalibur),
             "EXODUS" => Ok(Server::Exodus),
             "FAMFRIT" => Ok(Server::Famfrit),
             "HYPERION" => Ok(Server::Hyperion),
             "LAMIA" => Ok(Server::Lamia),
             "LEVIATHAN" => Ok(Server::Leviathan),
-            "MALBORO" => Ok(Server::Malboro),
             "ULTROS" => Ok(Server::Ultros),
+
+            // Crystal
+            "BALMUNG" => Ok(Server::Balmung),
+            "BRYNHILDR" => Ok(Server::Brynhildr),
+            "COEURL" => Ok(Server::Coeurl),
+            "DIABOLOS" => Ok(Server::Diabolos),
+            "GOBLIN" => Ok(Server::Goblin),
+            "MALBORO" => Ok(Server::Malboro),
+            "MATEUS" => Ok(Server::Mateus),
+            "ZALERA" => Ok(Server::Zalera),
+            
+            // dynamis
+            "HALICARNASSUS" => Ok(Server::Halicarnassus),
+            "MADUIN" => Ok(Server::Madiun),
+            "MARILITH" => Ok(Server::Marilith),
+            "SERAPH" => Ok(Server::Seraph),
+            "CUCHULAINN" => Ok(Server::Cuchulainn),
+            "KRAKEN" => Ok(Server::Kraken),
+            "RAFFLESIA" => Ok(Server::Rafflesia),
+            "GOLEM" => Ok(Server::Golem),
+
+            /// EU
             //  Chaos
             "CERBERUS" => Ok(Server::Cerberus),
             "LOUISOIX" => Ok(Server::Louisoix),
             "MOOGLE" => Ok(Server::Moogle),
             "OMEGA" => Ok(Server::Omega),
+            "Phantom" => Ok(Server::Phantom),
```

Comment:
Is this really supposed to be title case here? (Same for the fmt display for Server).